server/src/main/kotlin/suwayomi/tachidesk/manga/impl/Manga.kt

```diff
@@ -151,14 +155,20 @@ object Manga {
         val fileName = mangaId.toString()
 
         return getCachedImageResponse(saveDir, fileName) {
-            getManga(mangaId) // make sure is initialized
-
             val mangaEntry = transaction { MangaTable.select { MangaTable.id eq mangaId }.first() }
 
             val sourceId = mangaEntry[MangaTable.sourceReference]
             val source = getHttpSource(sourceId)
 
-            val thumbnailUrl = mangaEntry[MangaTable.thumbnail_url]!!
+            val thumbnailUrl: String = mangaEntry[MangaTable.thumbnail_url]
```

Comment:
Is there a specific reason why we need a thumbnail url, Tachi allows null urls without issue

Reply:
thumbnail url can absolutely be null, that NPE is for triggering a HTTP 404 in javalin.

Reply:
That works then 👍